customisation/language-identification.md

```diff
@@ -0,0 +1,16 @@
+---
+grand_parent: Building and research
+parent: Customisation
+title: Language identification
+description: Identifying the language of a text
+---
+
+**Language identification** tools identify the language of an unknown text.
+In machine translation, they are used in building monolingual [training data](/customisation/training-data.md) from large collections of unidentified texts such as webpages.
+They are also used to automatically identify the source language in many machine translation systems when the source language is unspecified.
+
+## Challenges
+
+- Similar languages may be mistaken for one another, for example Dutch and Afrikaans.
+- Language identification is more challenging on short pieces of text.
```

Comment:
Perhaps we can extend this sentence explaining why.
 "Language identification is more challenging on short pieces of text because many languages can include the same word, and..."